components/match2/commons/brkts_wiki_specific_base.lua

```diff
@@ -71,9 +71,13 @@ Called from MatchGroup/Display
 -- @returns module
 ]]
 function WikiSpecificBase.getMatchGroupContainer(matchGroupType)
-	return matchGroupType == 'matchlist'
-		and Lua.import('Module:MatchGroup/Display/Matchlist', {requireDevIfEnabled = true}).MatchlistContainer
-		or Lua.import('Module:MatchGroup/Display/Bracket', {requireDevIfEnabled = true}).BracketContainer
+	if matchGroupType == 'matchlist' then
+		return Lua.import('Module:MatchGroup/Display/Matchlist', {requireDevIfEnabled = true}).MatchlistContainer
+	elseif matchGroupType == 'singleMatch' then
+		return Lua.import('Module:MatchGroup/Display/SingleMatch', {requireDevIfEnabled = true}).SingleMatchContainer
+	else
+		return Lua.import('Module:MatchGroup/Display/Bracket', {requireDevIfEnabled = true}).BracketContainer
+	end
 end
```

Comment:
If we anticipate more options to show individual matches in the future, then one approach is to split this into `WikiSpecificBase.getMatchSummaryContainer`:

```
function WikiSpecificBase.getMatchSummaryContainer(displayMode)
	if displayMode == 'popup' then
		-- Original match summary inside popup (popup context must be provided by caller)
		return Lua.import('Module:MatchSummary', {requireDevIfEnabled = true}).MatchSummaryContainer

	elseif displayMode == 'singleMatch' then
		-- Single match, displayed flat on a page, embedded style
		return Lua.import('Module:MatchGroup/Display/SingleMatch', {requireDevIfEnabled = true}).SingleMatchContainer

	elseif displayMode == 'bigMatch' then
		-- Uses the whole page to display the match
		return Lua.import('Module:BigMatch/Display', {requireDevIfEnabled = true}).BigMatchContainer

	elseif displayMode == '2se' then
		-- Displays the match as a 2se bracket. The match summary is shown in a popup
		return Lua.import('Module:MatchGroup/Display/2seMatch', {requireDevIfEnabled = true}).MatchContainer

	end
end
```

Reply:
imo we only need the SingleMatch (+BigMatch on some wikis) in addition to the ones we currently have

Reply:
Good idea, since the required parameters are different between the singleMatch (and any future displays) and the existing matchGroups.